old/input-test.html

```diff
@@ -0,0 +1,32 @@
+<link rel="import" href="../polymer/polymer.html">
+
+<dom-module id="input-test">
+
+  <template>
+
+    <input value="{{hostValue::input}}">
```

Comment:
I really like this syntax for binding to native element updates. Just commenting :rainbow: